portable/GCC/ARM_CR5/port.c

```diff
@@ -74,25 +74,43 @@
     #error configMAX_API_CALL_INTERRUPT_PRIORITY must be greater than ( configUNIQUE_INTERRUPT_PRIORITIES / 2 )
 #endif
 
-/* Some vendor specific files default configCLEAR_TICK_INTERRUPT() in
- * portmacro.h. */
+#if configUSE_TASK_FPU_SUPPORT == 0
+    #ifdef __ARM_FP
+        #error __ARM_FP is defined, so configUSE_TASK_FPU_SUPPORT must be set to either to 1 or 2.

```

Comment:
Do we need to make clear that `__ARM_FP` is defined, when target is being compiled with `-mfpu=...`

Reply:
I added a comment above this line for clarity.